explainer.md

```diff
@@ -180,70 +180,73 @@ Media Capabilities should offer a means of surfacing when different MediaDecodin
-
-*   Colour gamut: HDR content requires a wider colour gamut than SDR content. Most screens use the sRGB colour gamut but p3 or BT.2020 are colour gamut that would usually be expected for HDR content.
-*   Colour/pixel depth: even if the screen has a wide colour gamut, the pixels need to be encoded in 30 bits (10 bits per colour component) instead of the usual 24 bits (8 bits per colour component). Otherwise, even if wider colour can be represented, precision would be lost.
-*   Brightness: because of darker blacks and brighter whites, a screen needs to have a large contrast ratio in order to be used for HDR content.
-
-#### Colour gamut
-
-The colour gamut of the screen could be exposed on the *Screen* interface. It is already part of the work in progress [CSS Media Queries 4](https://drafts.csswg.org/mediaqueries-4/#color-gamut) but because various information will have to be read from the *Screen* object for HDR content, it would make sense to have all of them grouped together.
+For HDR support detection, there are three main components whose capabilities need to be surfaced -- the decoder, renderer, and display. The decoder takes in an encoded stream and produces a decoded stream understood by the renderer, which in turn maps the stream's signals to those the display can properly output. Most of the time, the decoder and renderer are part of the UA while the display intuitively describes the physical output monitor, whether this be a computer monitor or TV. To match this natural modularity between the UA and the display, this API is compartmentalized into two (2) parts:
 
-#### Colour/Pixel Depth
+*   *MediaCapabilities.decodingInfo()*: handles the UA pieces, namely decoding and rendering. 
+*   TODO: various aspects of the display are being discussed. 
 
-It is already exposed on the *Screen* object but only for compatibility reasons. The [CSSOM View Module](https://www.w3.org/TR/cssom-view-1/#dom-screen-pixeldepth) should be updated or amended to make this information available.
-
-#### Brightness
-
-The minimum and maximum brightness should be exposed on the *Screen* object. In order to know the effective brightness, a website would need to know the brightness of the room which can be achieved with the [Ambient Light Sensor](https://w3c.github.io/ambient-light/).
-
-#### Example
+### Screen capabilities
 
-```JavaScript
-function canDisplayMyHDRStreams() {
-  // The conditions below are entirely made up :)
-  return window.screen.colorGamut == "rec2020" &&
-         window.screen.pixelDepth == "30" &&
-         window.screen.brightness.max > 500 &&
-         window.screen.brightness.min < 0.1;
-}
-```
+Even if a device is able to decode HDR content, if the screen isn’t able to show this content appropriately, it might not be worth using HDR content for the website because of the higher bandwidth consumptions but also the rendering might be worse than a SDR optimised video.
```

Comment:
Suggest simplifying text, e.g.:

When the UA reports ability to decode HDR content, but the screen does not report ability to render HDR, it is not recommended to transmit HDR content because it wastes bandwidth and may result in poorer actual rendering on some devices when compared to SDR optimized video.